### ISSUE TITLE
inlineToolbar config is invalid for paragraph

### ISSUE THREAD

**Issue by Seaony** [codex-team/editor.js#771]
Like this:

```js
  this.editor = new EditorJS({
      /**
       * Id of Element that should contain Editor instance
       */
      holderId: 'editor',
      tools: {
        paragraph: {
          class: Paragraph,
          inlineToolbar: ['bold'],
          config: {
            placeholder: 'Enter text...'
          }
        },
        header: {
          class: Header,
          inlineToolbar: false,
          config: {
            placeholder: 'Enter a header'
          }
        }
      }
    })
```

then:
![image](https://user-images.githubusercontent.com/29486364/57987438-e9b1f780-7ab3-11e9-99db-ee73f4ec2bed.png)

InlineToolbar seems to have lost its effect in the paragraph because I am misconfigured?

**Comment by gohabereg** [comment]
Hi @Seaony 
Will be resolved in 2.14 release

**Comment by Seaony** [comment]
Thanks!
